lib/word_2_quiz/answer.rb

```diff
@@ -8,23 +10,24 @@ def initialize(text = "", correct = false)
     end
 
     def self.from_paragraphs(paragraphs, solution)
-      text = paragraphs.map(&:to_html).join("\n")
-      answer_letter = paragraphs[0].text.match(/^[a-z]/)
+      non_empty_paragraphs = Helpers.strip_blanks(paragraphs)
+      text = non_empty_paragraphs.map(&:to_html).join("\n")
+      answer_letter = non_empty_paragraphs[0].text.match(/^[a-z]/)
```

Comment:
The only way to do that would be to write a helper that uses the index to get the letter and converts it into a hash. It will always be the first element, so it could also use `.first` instead of the zero element. I just realised it could use `start_with?` instead of match too. Paragraphs is an array of nokogiri nodes wrapped by the the docx gem, so not using indexes somewhere is difficult.